src/notifications/warning-message.js

```diff
@@ -22,5 +22,10 @@ export default {
 
     cannotFindConfigurationFile: 'Unable to find the "{path}" configuration file. Error details:\n' +
                                  '\n' +
-                                 '{err}'
+                                 '{err}',
+
+    problematicPathPatternPlaceholderForVideoRecording: '{placeholderList} path pattern placeholder{suffix} {verb} not suitable for the video recording\'s "pathPattern" option.\n' +
+                                                        '\n' +
+                                                        '{pronoun} value{suffix} will be replaced with an empty string.'
```

Comment:
Dirk's version:

```js
    problematicPathPatternPlaceholderForVideoRecording: 'The {placeholderList} path pattern placeholder{suffix} cannot be applied to the recorded video.\n' +
                                                        '\n' +
                                                        'The placeholder{suffix} {verb} replaced with an empty string.'
```

where `{suffix}` is `s` and `{verb}` is either `was` or `were`.